src/controllers/controller.polarArea.js

```diff
@@ -133,6 +138,14 @@ module.exports = function(Chart) {
 
 			meta.count = me.countVisibleElements();
 
+			var start = opts.startAngle || 0;
+			for (var i = 0; i < meta.count; i++) {
```

Comment:
Are you sure we only want to iterate until `meta.count` representing the number of visible elements, which are not necessary the first elements?